capstone/src/main/java/com/google/step/servlets/RemoteServlet.java

```diff
@@ -0,0 +1,16 @@
+package com.google.step.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/remote-api")
```

Comment:
Why is this needed? A little documentation can go a long way in this case.

Reply:
Haha, turns out it's not needed. It was in the example I was following so I thought it was necessary but it isn't! Deleted.